localization/languages/hu.json

```diff
@@ -69,10 +69,10 @@
     "newTask": "Új feladat", //"new" is a verb - it is used for a button at the bottom of the task overlay
     "defaultTaskName": "Feladat %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
     "taskDeleteWarning": {
-      "unsafeHTML": null //missing translation
+      "unsafeHTML": "Nem biztonságos HTML fájl" //missing translation
     },
     "tasksSearchTabs": null, //missing translation
-    "returnToTask": null, //missing translation
+    "returnToTask": "Vissza a feladathoz", //missing translation
```

Comment:
Can you confirm this is correct? I would expect it to specify "previous task"

English is:
```
    "returnToTask": "Return to your previous task",
```

Reply:
It is correct, I just missed the "previous" (elöző) word, so I'll fix it along with the others.
And thanks for the comments, It helps a lot.